datafusion/core/tests/sql/aggregates.rs

```diff
@@ -221,7 +221,7 @@ async fn csv_query_stddev_6() -> Result<()> {
 }
 
 #[tokio::test]
-async fn csv_query_median_1() -> Result<()> {
```

Comment:
If possible, I would recommend adding a basic test in sql for a median for all the different data types that are supported  (not just on aggregate_test_100 but a dedicated test setup with known data (maybe integers 10, 9, 8, ... 0)

Reply:
Added in https://github.com/apache/arrow-datafusion/pull/3009/commits/ef1effdbfcafb9147184aa59dfb52ffcbe7a572d